internal/pkg/build/files/files.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package files
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func expandPath(path string) ([]string, error) {
+	var output, stderr bytes.Buffer
+	cmd := exec.Command("/bin/sh", "-c", "echo "+path)
```

Comment:
Won't filepath.Glob() not work in this case? It resolves wildcards and returns files in a list of strings.

Reply:
Unfortunately for this application it doesn't. All the globbing related go packages I have found do not handle hidden files in the same manner as most shells, which is how users will expect these wildcards to work.

Reply:
At least on Mac, I see that filepath.Glob() returns hidden files.